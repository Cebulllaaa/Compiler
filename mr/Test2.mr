RESET b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
INC a
SWAP c
SWAP c
STORE b
RESET b
INC b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
INC a
SHIFT c
SWAP c
SWAP c
STORE b
RESET b
INC b
INC b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
INC a
SHIFT c
INC a
SWAP c
SWAP c
STORE b
RESET b
INC b
INC b
INC b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
SHIFT c
INC a
SHIFT c
SHIFT c
SWAP c
SWAP c
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
RESET c
LOAD c
SWAP c
RESET d
INC d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD c
JPOS 7
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JZERO 17
RESET a
ADD c
SHIFT g
SHIFT h
SUB c
JZERO 4
SWAP e
ADD d
SWAP e
SWAP c
SHIFT g
SWAP c
SWAP d
SHIFT h
SWAP d
JUMP -18
SWAP e
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
INC d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD c
JPOS 7
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JZERO 17
RESET a
ADD c
SHIFT g
SHIFT h
SUB c
JZERO 4
SWAP e
ADD d
SWAP e
SWAP c
SHIFT g
SWAP c
SWAP d
SHIFT h
SWAP d
JUMP -18
SWAP e
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
INC d
INC d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD c
JPOS 7
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JZERO 17
RESET a
ADD c
SHIFT g
SHIFT h
SUB c
JZERO 4
SWAP e
ADD d
SWAP e
SWAP c
SHIFT g
SWAP c
SWAP d
SHIFT h
SWAP d
JUMP -18
SWAP e
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SHIFT b
SWAP b
RESET c
INC c
INC c
SWAP c
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
INC a
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
RESET a
INC a
SHIFT d
SHIFT d
SHIFT d
SWAP d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD c
JPOS 7
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JZERO 17
RESET a
ADD c
SHIFT g
SHIFT h
SUB c
JZERO 4
SWAP e
ADD d
SWAP e
SWAP c
SHIFT g
SWAP c
SWAP d
SHIFT h
SWAP d
JUMP -18
SWAP e
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
LOAD b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
INC a
SWAP c
LOAD c
SWAP c
SWAP b
SUB c
JNEG 466
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
INC a
SWAP b
RESET c
SWAP c
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
RESET a
INC a
SHIFT d
SHIFT d
SHIFT d
SWAP d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD d
JZERO 60
RESET f
JPOS 8
INC f
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JPOS 6
INC f
INC f
RESET a
SUB c
SWAP c
RESET a
ADD c
SUB d
JNEG 6
SWAP d
SHIFT h
SWAP d
INC e
JUMP -8
RESET a
ADD e
JPOS 2
JUMP 11
RESET a
SWAP d
SHIFT g
SWAP d
ADD c
SUB d
JNEG 2
SWAP c
DEC e
JUMP -13
RESET a
ADD f
SHIFT g
JZERO 6
RESET a
SUB c
JZERO 3
ADD d
SWAP d
RESET a
ADD f
SHIFT g
SHIFT h
SUB f
JZERO 6
RESET a
SUB c
SWAP c
JUMP 2
RESET c
SWAP c
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
SWAP b
LOAD b
SWAP b
RESET c
SWAP b
SUB c
JZERO 2
JUMP 213
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
RESET a
INC a
SHIFT d
SHIFT d
SHIFT d
SWAP d
LOAD d
SWAP d
RESET g
RESET h
INC h
RESET e
RESET a
ADD d
JZERO 55
JPOS 7
RESET a
SUB d
SWAP d
RESET a
SUB c
SWAP c
RESET a
SWAP f
RESET a
ADD c
JPOS 4
INC f
RESET a
SUB c
SWAP c
RESET a
ADD c
SUB d
JNEG 6
SWAP d
SHIFT h
SWAP d
INC g
JUMP -8
SWAP g
JZERO 20
SWAP g
DEC g
SWAP e
SHIFT h
SWAP e
SWAP d
DEC h
DEC h
SHIFT h
INC h
INC h
SWAP d
RESET a
ADD c
SUB d
JNEG 3
SWAP c
INC e
JUMP -20
SWAP f
JZERO 7
RESET a
SUB e
SWAP e
SWAP c
JZERO 2
DEC e
SWAP e
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
INC a
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
INC a
SHIFT c
INC a
SWAP c
LOAD c
SWAP c
RESET d
INC d
SWAP c
ADD d
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
RESET a
INC a
SHIFT d
SHIFT d
SHIFT d
SWAP d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD d
JZERO 60
RESET f
JPOS 8
INC f
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JPOS 6
INC f
INC f
RESET a
SUB c
SWAP c
RESET a
ADD c
SUB d
JNEG 6
SWAP d
SHIFT h
SWAP d
INC e
JUMP -8
RESET a
ADD e
JPOS 2
JUMP 11
RESET a
SWAP d
SHIFT g
SWAP d
ADD c
SUB d
JNEG 2
SWAP c
DEC e
JUMP -13
RESET a
ADD f
SHIFT g
JZERO 6
RESET a
SUB c
JZERO 3
ADD d
SWAP d
RESET a
ADD f
SHIFT g
SHIFT h
SUB f
JZERO 6
RESET a
SUB c
SWAP c
JUMP 2
RESET c
SWAP c
STORE b
JUMP -226
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
INC a
SWAP b
LOAD b
SWAP b
RESET c
SWAP b
SUB c
JPOS 2
JUMP 27
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SHIFT b
SWAP b
LOAD b
SWAP b
SWAP b
PUT
RESET b
INC b
RESET a
INC a
SHIFT b
INC a
SHIFT b
INC a
SWAP b
LOAD b
SWAP b
SWAP b
PUT
JUMP 87
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SHIFT b
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
SWAP c
ADD d
STORE b
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
INC a
SWAP b
RESET c
INC c
RESET a
INC a
SHIFT c
SHIFT c
SHIFT c
SWAP c
LOAD c
SWAP c
RESET d
INC d
RESET a
INC a
SHIFT d
SHIFT d
SHIFT d
SWAP d
LOAD d
SWAP d
RESET g
DEC g
RESET h
INC h
RESET e
RESET a
ADD c
JPOS 7
RESET a
SUB c
SWAP c
RESET a
SUB d
SWAP d
RESET a
ADD c
JZERO 17
RESET a
ADD c
SHIFT g
SHIFT h
SUB c
JZERO 4
SWAP e
ADD d
SWAP e
SWAP c
SHIFT g
SWAP c
SWAP d
SHIFT h
SWAP d
JUMP -18
SWAP e
STORE b
JUMP -486
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
LOAD b
SWAP b
RESET c
INC c
SWAP b
SUB c
JZERO 17
RESET b
INC b
RESET a
INC a
SHIFT b
SHIFT b
SWAP b
LOAD b
SWAP b
SWAP b
PUT
RESET b
INC b
SWAP b
PUT
JUMP 1
HALT
