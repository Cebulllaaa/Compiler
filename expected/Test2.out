> 8
> 3
> 25
> 7
> 13
> 41
> 61
> 641
> 1321
> 6700417
> 613566757
> 715827883